kbase-extension/static/kbase/js/widgets/function_output/kbaseGenericSetViewer.js

```diff
@@ -48,26 +48,32 @@ define([
             this._super(options);
```

Comment:
These changes are back! 😱 
Was that intentional this time?

Reply:
Nope! Grrrrrr...